devel/cctools/Portfile

```diff
@@ -88,45 +86,81 @@ if {![some_llvm_variant_set]} {
     #     default_variants +llvm33
     # }
 
-    if {![some_llvm_variant_set] && ${os.major} >= 10} {
+    if {${os.major} >= 18} {
+        default_variants +xcode
+    } elseif {![some_llvm_variant_set] && ${os.major} >= 10} {
         default_variants +llvm50
     }
+
+}
+
+variant xcode description "Use Xcode supplied toolkit" { }
+if {[variant_isset xcode]} {
+   patch { }
+   extract { }
+   fetch { }
+   build { }
+   pre-destroot { }
+   post-destroot { }
+   post-extract { }
+   checksum { }
+   destroot {
+       set xcode_cmds {bitcode-strip nm as ar otool ranlib lipo libtool segedit strip size size-classic strings}
+       foreach xcode_cmd $xcode_cmds {
+           set mp_cmd ${destroot}${prefix}/bin/${xcode_cmd}
+           # hardcode link to command line tools version
+           #ln -s /usr/bin/${xcode_cmd} ${mp_cmd}
```

Comment:
Might this want a separate variant (e.g. `xcode_cmd`)? I understand that it is commented out to help one use this port as a base for using just the command line tools, yet, just leaving this commented out will require one to edit files nevertheless, and is kind of pointless.